content/guides/_index.md

```diff
@@ -36,6 +36,9 @@ comments: false
   
 {{< /cards >}}
 
+## Static Site Generator (SSG)
+{{< readfile file="guides/cards-static.md" >}}
```

Comment:
Simply putting the content of `/cards-static.md` should be enough, do we need a partial for it?.